x-pack/plugins/security_solution/public/management/components/management_tabs.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { AdministrationSubTab } from '../types';
+import { SecurityPageName } from '../../app/types';
+import { NavTab } from '../../common/components/navigation/types';
+
+export const managementTabs: Record<string, NavTab> = {
+  [AdministrationSubTab.endpoints]: {
+    name: i18n.translate('xpack.securitySolution.managementTabs.hosts', {
+      defaultMessage: 'Hosts',
+    }),
+    id: AdministrationSubTab.endpoints,
+    href: '/hosts',
```

Comment:
I believe this was all just renamed from Hosts to Endpoints (url too).

Reply:
we should use the `getEndpointListPath()` method to get path along with `formatUrl()` if we need a full href

Reply:
Fixed